source/class/qx/tool/cli/commands/Command.js

```diff
@@ -334,6 +335,9 @@ qx.Class.define("qx.tool.cli.commands.Command", {
      * @return {Boolean}
      */
     isExplicitArg(option) {
+      /**
+       * @param option
+       */
```

Comment:
imho private functions like this do not need comments unless there is a real need for it - and adding a blank comment is not really helpful, it just adds cruft.